flink-table/flink-sql-client/src/main/resources/META-INF/NOTICE

```diff
@@ -9,3 +9,4 @@ See bundled license files for details.
 
 - org.jline:jline-terminal:3.9.0
 - org.jline:jline-reader:3.9.0
+- com.ibm.icu:icu4j:65.1
```

Comment:
@KurtYoung @leonardBang I don't think icu is a BSD license. Could you please fix this, or revert the changes.

Reply:
I checked https://www.apache.org/legal/resolved.html, and ICU is in category A, isn't is sufficient?

Reply:
The license is ok to add, yes. But in this list it's listed under the `BSD` licenses. Icu is not licensed under a BSD license. See for example https://github.com/apache/flink/blob/4035f03a1de24b7e292466e9fa0a0de9cb10f358/flink-dist/src/main/resources/META-INF/NOTICE#L20, which has many different blocks for different licenses.

Reply:
I see what you mean, thanks